src/views/process-panel/process-information-card.tsx

```diff
@@ -114,11 +115,16 @@ export const ProcessInformationCard = withStyles(styles, { withTheme: true })(
                 <Grid container>
                     <Grid item xs={6}>
                         <DetailsAttribute classLabel={classes.label} classValue={classes.value}
-                            label='From' value={process.container ? formatDate(process.container.startedAt!) : 'N/A'} />
+                                          label='From'
+                                          value={process.container ? formatDate(process.container.startedAt!) : 'N/A'} />
                         <DetailsAttribute classLabel={classes.label} classValue={classes.value}
-                            label='To' value={process.container ? formatDate(process.container.finishedAt!) : 'N/A'} />
-                        <DetailsAttribute classLabel={classes.label} classValue={classes.link}
-                            label='Workflow' value='???' />
+                                          label='To'
+                                          value={process.container ? formatDate(process.container.finishedAt!) : 'N/A'} />
```

Comment:
Do not use exclamation mark to cheat on the type system. Calculate value earlier  with something like so that TypeScript will infer whether container is defined or not:
```
const { container } = process;
const finishedAt = container ? formatDate(container.finishedAt) : 'N/A'

...
value={finishedAt}
...
```

Reply:
actually it wasn't me :) I just reformatted that code, but ok, I can change it anyway